db/migrate/20230228085212_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] The user foreign key for other tables should be as `author_id` not as `user_id` to follow the entities as on the ERD diagram. Please make sure you follow this and I would suggest looking at the below code for some hints.

```ruby
add_reference :comments, :author, null: false, foreign_key: {to_table: 'users'}

Reply:
Thanks!